packages/fiori/src/Wizard.ts

```diff
@@ -576,34 +587,32 @@ class Wizard extends UI5Element {
 		// using the "data-ui5-wizard-after-current-tab" and "data-ui5-wizard-expanded-tab-prev" attributes
 		for (let i = 0; i < tabs.length; i++) {
 			if (tabs[i].getAttribute(EXPANDED_STEP) === "true" && tabs[i - 1] && tabs[i - 1].getAttribute(EXPANDED_STEP) === "false") {
-				tabs[i - 1].setAttribute(BEFORE_EXPANDED_STEP, true);
+				tabs[i - 1].setAttribute(BEFORE_EXPANDED_STEP, "true");
 			}
 
 			if (tabs[i].getAttribute(EXPANDED_STEP) === "false" && tabs[i - 1] && tabs[i - 1].getAttribute(EXPANDED_STEP) === "true") {
-				tabs[i].setAttribute(AFTER_EXPANDED_STEP, true);
+				tabs[i - 1].setAttribute(BEFORE_EXPANDED_STEP, "true");
 				break;
 			}
 		}
 	}
 
-	_isGroupAtStart(selectedStep) {
+	_isGroupAtStart(selectedStep: WizardTab) {
 		const iStepNumber = this.stepsInHeaderDOM.indexOf(selectedStep);
-
 		return selectedStep.getAttribute(EXPANDED_STEP) === "false" && selectedStep.getAttribute(BEFORE_EXPANDED_STEP) === "true" && iStepNumber > 0;
 	}
 
-	_isGroupAtEnd(selectedStep) {
+	_isGroupAtEnd(selectedStep: WizardTab) {
 		const iStepNumber = this.stepsInHeaderDOM.indexOf(selectedStep);
-
 		return selectedStep.getAttribute(EXPANDED_STEP) === "false" && selectedStep.getAttribute(AFTER_EXPANDED_STEP) === "true" && (iStepNumber + 1 < this.steps.length);
 	}
 
-	async _showPopover(oDomTarget, bAtStart) {
-		const tabs = Array.from(this.shadowRoot.querySelectorAll("[ui5-wizard-tab]"));
+	async _showPopover(oDomTarget: WizardTab, isAtStart: boolean) {
+		const tabs: Array<WizardTab> = Array.from(this.shadowRoot!.querySelectorAll("[ui5-wizard-tab]"));
```

Comment:
Array.from(this.shadowRoot!.querySelectorAll<WizardTab>("[ui5-wizard-tab]"));

Reply:
I dont get this